po/clubhouse.pot

```diff
@@ -0,0 +1,175 @@
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: data/achievement-summary-view.ui:33
+msgid "Achievement Badge Name"
+msgstr ""
+
+#: data/achievement-summary-view.ui:49
+msgid "Description"
+msgstr ""
+
+#: data/achievements-view.ui:66
+msgid "Achievements"
+msgstr ""
+
+#: data/achievements-view.ui:131
+msgid "page0"
+msgstr ""
+
+#: data/achievements-view.ui:163
+msgid "page1"
+msgstr ""
+
+#: data/activity-card.ui:219
+msgid "play"
+msgstr ""
+
+#: data/character-view.ui:81 data/clubhouse-window.ui:255
+#: data/clubhouse-window.ui:306
+msgid "Clubhouse"
+msgstr ""
+
+#: data/character-view.ui:124
+msgid "Workspace"
+msgstr ""
+
+#: data/clubhouse-view-main-layer.ui:102
+msgid "Hack extension is not enabled. Click here to solve"
+msgstr ""
+
+#: data/clubhouse-view-main-layer.ui:115
+msgid ""
+"The Hack GNOME Shell Extension is not installed. <b>Install extension</b>."
+msgstr ""
+
+#: data/clubhouse-view-main-layer.ui:133
+msgid "Installing hack extension…"
+msgstr ""
+
+#: data/clubhouse-window.ui:41 eosclubhouse/clubhouse.py:108
+msgid "Games"
+msgstr ""
+
+#: data/clubhouse-window.ui:87 eosclubhouse/clubhouse.py:102
+msgid "Art"
+msgstr ""
+
+#: data/clubhouse-window.ui:133 eosclubhouse/clubhouse.py:126
+msgid "Web"
+msgstr ""
+
+#: data/clubhouse-window.ui:179 eosclubhouse/clubhouse.py:120
+msgid "Maker"
+msgstr ""
+
+#: data/clubhouse-window.ui:225
+msgid "Operating System"
+msgstr ""
+
+#: data/clubhouse-window.ui:342
+msgid "Pathways"
+msgstr ""
+
+#: data/clubhouse-window.ui:369
+msgid "Hack News"
+msgstr ""
+
+#: data/com.hack_computer.Clubhouse.desktop:4
+#: data/com.hack_computer.Clubhouse.metainfo.xml:7
+msgid "Hack"
+msgstr ""
+
+#: data/com.hack_computer.Clubhouse.desktop:5
+msgid "eos-clubhouse"
+msgstr ""
+
+#: data/com.hack_computer.Clubhouse.desktop:6
+msgid "Hack Clubhouse application"
+msgstr ""
+
+#: data/com.hack_computer.Clubhouse.desktop:7
+msgid "com.hack_computer.Clubhouse"
+msgstr ""
+
+#: data/com.hack_computer.Clubhouse.desktop:12
+msgid "hack;learn;code;programming;game;clubhouse;"
+msgstr ""
+
+#: data/com.hack_computer.Clubhouse.metainfo.xml:21
+msgid "The destination for kids to learn how to code"
+msgstr ""
+
+#: data/com.hack_computer.Clubhouse.metainfo.xml:23
+msgid ""
+"Kids learn how to code, develop creative problem-solving skills and build "
+"confidence in a fun and safe environment."
+msgstr ""
+
+#: data/com.hack_computer.Clubhouse.metainfo.xml:30
+msgid "The main window with a quest running"
+msgstr ""
+
+#: data/com.hack_computer.Clubhouse.metainfo.xml:34
+msgid "The game pathway"
+msgstr ""
+
+#: data/com.hack_computer.Clubhouse.metainfo.xml:38
+msgid "The game pathway with a quest running"
+msgstr ""
+
+#: data/com.hack_computer.Clubhouse.metainfo.xml:42
+msgid "The main window"
+msgstr ""
+
+#: data/com.hack_computer.Clubhouse.metainfo.xml:51
+msgid "The Hack Team"
+msgstr ""
+
+#: data/message.ui:38
+msgid "icon-down"
```

Comment:
I guess this is the title for the button to move the dialogue down? If so we should edit the `message.ui` file to something better. Maybe "Move dialogue down". Otherwise this will confuse translators. Also for accesibility.